### ISSUE TITLE
initial start-up

### ISSUE THREAD

**Issue by manerotoni** [embl-cba/fiji-plugin-cats#11]
When you start CATS and then specify the output folder, CATS issues an error if you use a non-empty folder. Only when the target folder is empty or contains files of the appropriate format (e.g. _classified--C01--T000001--Z000001.tif_ ...) does CATS starts.

  
The messages are

``Error: could not open data set. There needs to be at least one file matching the naming scheme.``

and 

`` The module _de.embl.cba.cats.ui.CATSCommand_ failed to initialize.``

I uploaded the image file to 

https://owncloud.gwdg.de/index.php/s/xiGLOIDV4yhtSkD

Ciao
Antonio

**Comment by tischi** [comment]
> When you start CATS and then specify the output folder, CATS issues an error if you use a non-empty folder. Only when the target folder is empty or contains files of the appropriate format (e.g. classified--C01--T000001--Z000001.tif ...) does CATS starts.

Yes...., this is kind of the intended behaviour, why would you like to point to a folder that contains other files?

**Comment by tischi** [comment]
Hi Sam,
Do you think I should change something here? Or can we close this issue?
